sdks/python/setup.py

```diff
@@ -237,7 +237,7 @@ def get_portability_package_data():
               'Sphinx>=1.5.2,<2.0',
               # Pinning docutils as a workaround for Sphinx issue:
               # https://github.com/sphinx-doc/sphinx/issues/9727
-              'docutils==0.17.1'
+              'docutils==0.19'
```

Comment:
I am not certain if the upgrade will cause any issues w.r.t. the above comment. We should test it or dig up some commit history to see what went wrong; then, we can remove the comment if no longer an issue.

Reply:
Agreed, who can take this?